apiserver/common/upgradeseries.go

```diff
@@ -152,6 +152,44 @@ func (u *UpgradeSeriesAPI) SetUpgradeSeriesStatus(args params.SetUpgradeSeriesSt
 	return u.setUpgradeSeriesStatus(args)
 }
 
+func (u *UpgradeSeriesAPI) CompleteUnitUpgradeSeries(args params.SetUpgradeSeriesStatusParams) (params.ErrorResults, error) {
```

Comment:
This seems to actually be doing 
SetUnitsUpgradeSeriesStatus(), not completing the upgrade series.